cfn-resources/cluster/docs/README.md

```diff
@@ -139,6 +141,16 @@ _Allowed Values_: <code>AWS</code> | <code>GCP</code> | <code>AZURE</code> | <co
 
 _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
+#### GlobalClusterSelfManagedSharding
+
+Flag that indicates if cluster uses Atlas-Managed Sharding (false) or Self-Managed Sharding (true).
```

Comment:
same, as this is the only doc, I'll probably go for the long explanation (the one in the TF resource)